app/components/HelpGrid/index.tsx

```diff
@@ -9,46 +10,46 @@ const helpItems = [
       'For both technical and non-technical roles in AI alignment, governance, and field-building',
     impact: 'Highest direct impact',
     icon: Briefcase,
-    action: '/howcanihelp/career',
+    action: helpUrl('career'),
```

Comment:
this makes it a lot easier to change urls, or at least know that they are correct(ish)